platform/fio/client.go

```diff
@@ -2,8 +2,32 @@ package fio
 
 import (
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
 )
 
 type Client struct {
 	blockatlas.Request
 }
+
+type GetPubAddressRequest struct {
```

Comment:
Maybe it is ok to add structures to the model.go file like any other platform?
[example](https://github.com/trustwallet/blockatlas/blob/master/platform/cosmos/model.go)

Reply:
Done, thanks